packages/core/package.json

```diff
@@ -66,22 +66,21 @@
   },
   "dependencies": {
     "@sentry/babel-plugin-component-annotate": "3.1.2",
-    "@sentry/browser": "8.54.0",
+    "@sentry/browser": "9.1.0",
     "@sentry/cli": "2.42.1",
-    "@sentry/core": "8.54.0",
-    "@sentry/react": "8.54.0",
-    "@sentry/types": "8.54.0",
-    "@sentry/utils": "8.54.0"
+    "@sentry/core": "9.1.0",
+    "@sentry/react": "9.1.0",
+    "@sentry/types": "9.1.0"
   },
   "devDependencies": {
     "@babel/core": "^7.25.2",
     "@expo/metro-config": "0.19.5",
     "@mswjs/interceptors": "^0.25.15",
     "@react-native/babel-preset": "0.77.1",
-    "@sentry-internal/eslint-config-sdk": "8.54.0",
-    "@sentry-internal/eslint-plugin-sdk": "8.54.0",
-    "@sentry-internal/typescript": "8.54.0",
-    "@sentry/wizard": "3.42.0",
+    "@sentry-internal/eslint-config-sdk": "9.1.0",
+    "@sentry-internal/eslint-plugin-sdk": "9.1.0",
+    "@sentry-internal/typescript": "9.1.0",
+    "@sentry/wizard": "3.40.0",
```

Comment:
Should we keep `3.42.0` from the base branch?

Reply:
sounds  good to me! fixed on https://github.com/getsentry/sentry-react-native/pull/4568/commits/9a0b21ecf6b7f52098035b36837d41296502532f